### PR TITLE
Update apisec-scan.yml

### PR DESCRIPTION
**Pull request by SyedsImran**
## Pre-requisites

- [x] Prior to submitting a new workflow, please apply to join the GitHub Technology Partner Program: [partner.github.com/apply](https://partner.github.com/apply?partnershipType=Technology+Partner).

---

### **Please note that at this time we are only accepting new starter workflows for Code Scanning. Updates to existing starter workflows are fine.**

---

## Tasks

**For _all_ workflows, the workflow:**

- [x] Should be contained in a `.yml` file with the language or platform as its filename, in lower, [_kebab-cased_](https://en.wikipedia.org/wiki/Kebab_case) format (for example, [`docker-image.yml`](https://github.com/actions/starter-workflows/blob/main/ci/docker-image.yml)).  Special characters should be removed or replaced with words as appropriate (for example, "dotnet" instead of ".NET").
- [x] Should use sentence case for the names of workflows and steps (for example, "Run tests").
- [x] Should be named _only_ by the name of the language or platform (for example, "Go", not "Go CI" or "Go Build").
- [x] Should include comments in the workflow for any parts that are not obvious or could use clarification.
- [x] Should specify least privileged [permissions](https://docs.github.com/en/actions/security-guides/automatic-token-authentication#modifying-the-permissions-for-the-github_token) for `GITHUB_TOKEN` so that the workflow runs successfully.

**For _CI_ workflows, the workflow:**

- [x] Should be preserved under [the `ci` directory](https://github.com/actions/starter-workflows/tree/main/ci).
- [x] Should include a matching `ci/properties/*.properties.json` file (for example, [`ci/properties/docker-publish.properties.json`](https://github.com/actions/starter-workflows/blob/main/ci/properties/docker-publish.properties.json)).
- [x] Should run on `push` to `branches: [ $default-branch ]` and `pull_request` to `branches: [ $default-branch ]`.
- [x] Packaging workflows should run on `release` with `types: [ created ]`.
- [x] Publishing workflows should have a filename that is the name of the language or platform, in lower case, followed by "-publish" (for example, [`docker-publish.yml`](https://github.com/actions/starter-workflows/blob/main/ci/docker-publish.yml)).

**For _Code Scanning_ workflows, the workflow:**

- [x] Should be preserved under [the `code-scanning` directory](https://github.com/actions/starter-workflows/tree/main/code-scanning).
- [x] Should include a matching `code-scanning/properties/*.properties.json` file (for example, [`code-scanning/properties/codeql.properties.json`](https://github.com/actions/starter-workflows/blob/main/code-scanning/properties/codeql.properties.json)), with properties set as follows:
  - [x] `name`: Name of the Code Scanning integration.
  - [x] `creator`: Name of the organization/user producing the Code Scanning integration.
  - [x] `description`: Short description of the Code Scanning integration.
  - [x] `categories`: Array of languages supported by the Code Scanning integration.
  - [x] `iconName`: Name of the SVG logo representing the Code Scanning integration. This SVG logo must be present in [the `icons` directory](https://github.com/actions/starter-workflows/tree/main/icons).
- [x] Should run on `push` to `branches: [ $default-branch, $protected-branches ]` and `pull_request` to `branches: [ $default-branch ]`. We also recommend a `schedule` trigger of `cron: $cron-weekly` (for example, [`codeql.yml`](https://github.com/actions/starter-workflows/blob/c59b62dee0eae1f9f368b7011cf05c2fc42cf084/code-scanning/codeql.yml#L14-L21)).

**Some general notes:**

- [x] This workflow must _only_ use actions that are produced by GitHub, [in the `actions` organization](https://github.com/actions), **or**
- [x] This workflow must _only_ use actions that are produced by the language or ecosystem that the workflow supports.  These actions must be [published to the GitHub Marketplace](https://github.com/marketplace?type=actions).  We require that these actions be referenced using the full 40 character hash of the action's commit instead of a tag.  Additionally, workflows must include the following comment at the top of the workflow file:
    ```
    # This workflow uses actions that are not certified by GitHub.
    # They are provided by a third-party and are governed by
    # separate terms of service, privacy policy, and support
    # documentation.
    ```
- [x] Automation and CI workflows should not send data to any 3rd party service except for the purposes of installing dependencies.
- [x] Automation and CI workflows cannot be dependent on a paid service or product.

### CODE REVIEW

File: code-scanning/apisec-scan.yml
Comment (Phantsure) [inline code-scanning/apisec-scan.yml:13]: Would need to verify your identity before doc changes. 
@mattlovestech Can you please do that?

Comment (mohdrashid1) [reply code-scanning/apisec-scan.yml:13]: @Phantsure @mattlovestech it has been a week gone to marge my small commit. Please approve it ASAP as we have pending work on it

Comment (liamgutierrez77) [reply code-scanning/apisec-scan.yml:13]: T And also I'm sending this from my phone iPhone because my laptop is hacked. I sent this from my iPhone because I took a picture from my laptop phone Google Chrome. hese people are hacking me I don't care what you do just do whatever you have to do it because these people and would not stop hacking me over and over. I will send more pictures about these people look at in this area. I'll try many time to call taiga, CIA and this is again this is the location and also they hacking the gig hubs abs many times to prevent me.imageimage

Comment (liamgutierrez77) [reply code-scanning/apisec-scan.yml:13]: Take a look there, hockey prevent me to send informationimage

**Comment by liamgutierrez77** [comment]
i approval to all document from me you guys check

File: code-scanning/apisec-scan.yml
Comment (liamgutierrez77) [inline code-scanning/apisec-scan.yml:13]: aim send this form my iPhone this picture

**Review by liamgutierrez77 — APPROVED** [review]

File: code-scanning/apisec-scan.yml
Comment (liamgutierrez77) [reply code-scanning/apisec-scan.yml:13]: Lilliam Collado Gutierrez this my name 07/24/1977 my birthday

**Comment by SyedsImran** [comment]
waiting for this PR to be approved!

File: code-scanning/apisec-scan.yml
Comment (mattlovestech) [reply code-scanning/apisec-scan.yml:13]: @Phantsure it does not appear that API Sec is apart of the Technology Partnerships Program.

Comment (mohdrashid1) [reply code-scanning/apisec-scan.yml:13]: @Phantsure @mattlovestech What we need to do now APISec is already a part of TPP

Comment (Phantsure) [reply code-scanning/apisec-scan.yml:13]: > Prior to submitting a new workflow, please apply to join the GitHub Technology Partner Program: [partner.github.com/apply](https://partner.github.com/apply?partnershipType=Technology+Partner).

@mohdrashid1 This might help ⬆️

Comment (mohdrashid1) [reply code-scanning/apisec-scan.yml:13]: @Phantsure We have apply for GitHub Technology Partner Program. Please update us and approve PR

Comment (Phantsure) [reply code-scanning/apisec-scan.yml:13]: @mattlovestech Can you verify if they meet requirements?

**Comment by AllaithpPr** [comment]
21

**Comment by AllaithpPr** [comment]
zz

**Comment by AllaithpPr** [comment]
S

File: code-scanning/apisec-scan.yml
Comment (mohdrashid1) [reply code-scanning/apisec-scan.yml:13]: @mattlovestech @Phantsure r u guys done with checking our(apisec.ai) GTTP requirements.

Comment (mattlovestech) [reply code-scanning/apisec-scan.yml:13]: @mohdrashid1 can you send an email to partnerships@github.com to confirm that this PR was submitted by the APIsec team? After we get confirmation this can be approved.

Comment (SyedsImran) [reply code-scanning/apisec-scan.yml:13]: @mattlovestech   We sent an email from syedimran@apisec.ai email address to partnerships@github.com to confirm that this PR #1898 was by APIsec team.
Please review it and aprprove this PR.

Comment (mattlovestech) [reply code-scanning/apisec-scan.yml:13]: @Phantsure confirming an email was sent, authors are from apisec.ai

**Comment by Melihkarlar** [comment]
Bende şube ile güncel değil

**Review by Phantsure — APPROVED** [review]

**Pull request merged**
